docs/proposal-optional-output-extension.md

```diff
@@ -41,52 +30,33 @@ There are currently three problems with expiry (as of cylc-flow 8.1.4):
    This proposal does not address the late event problem and recommends this
    should be a documented limitation.
 
-
-## Background: Optional Outputs
-
-The SoD model, initially had implicit graph branching. This was a neat feature, however,
-meant that non-success cases were not properly caught so was not suitable behaviour for
-real-world use.
-
-The problem was that Cylc had not been provided with the required information to tell
-the difference between and "expected" or "permitted" outcome and a "problem" outcome.
-
-Consequently, optional outputs were introduced in order to provide Cylc with enough
-information to resolve these situations and stall, where appropriate, or continue/stop
-otherwise. This turned implicit branching explicit providing the runtime safety
-required for production use.
-
-Unfortunately, the optional outputs system doesn't have enough information at
-present to adequately handle expiry as an optional output as `:expire?` would
-imply `:succeeded?` and `:failed?`. This is a limitation of the optional output
-mechanism which this proposal aims to address by providing sufficient
-information to the scheduler to allow it to differentiate between "permitted"
-and "problem" outcomes.
-
-
 ## Proposal
 
 1. Expiry should remain implemented as a task output.
 
-   If a task is clock-expired OR if `expired` output is set manually, then the
-   task status should change to expired as per
-   [cylc set proposal](https://cylc.github.io/cylc-admin/proposal-new-output-syntax.html).
```

Comment:
This statement hasn't been revoked, just removed from the proposal as orthogonal.